src/controllers/task.ts

```diff
@@ -38,5 +39,15 @@ const addTask = (req: Request, res: Response, next: NextFunction) => {
     })
     .catch(() => next(new CustomError(500, 'server error')))
 };
+const deleteTaskById = (req: TokenRequest, res: Response, next: NextFunction) => {
+  const { taskId } = req.params;

```

Comment:
you get the taskId from the param, how you can access the params here? have you tried to print it?